MyObjects/Pages/Saucedemo/LoginPage.cs

```diff
@@ -0,0 +1,49 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyObjects.Pages.Saucedemo
+{
+    public class LoginPage
+    {
+        IWebDriver webDriver;
+
+        public readonly By txtUsername = By.XPath("//input[contains(@id,'user-name')]");
```

Comment:
it's great that you've mastered 'contains', but here it's unnecessary, you may simply replace with:
By.XPath("//input[@id='user-name']")
![image](https://user-images.githubusercontent.com/3397575/201656540-01ce7d16-98f7-4896-b2d6-23f041e29dde.png)